open-company/open-company-web
# Content visibility settings

Card: https://trello.com/c/tJjY9x26

Related PRs:
- [Storage](https://github.com/open-company/open-company-storage/pull/191)
- [Digest](https://github.com/open-company/open-company-digest/pull/21)

To test:
- get a public board URL, hold onto it
- get a public entry share URL, hold onto it
- get a secure link, hold onto it (via digest)
- go to settings
- turn on all the content visibility checkbox
- save
- load the org anonymously
- [x] does it not work?
- load the public board anonymously
- [x] does it not work?
- load the public entry anonymously
- [x] does it not work?
- load the secure link anonymously
- [x] does it not work?
- go to share a post
- [x] do you NOT see the public url share option? Good
- now try to add a new section
- [x] do you NOT see the public access option? Good
- now try to edit a section settings
- [x] do you NOT see the public access option? Good
- now request Slack digest
- [x] do you get simple links w/o id token in it? Good
- now request Email digest
- [x] do you get simple links w/o id token in it? Good
- now turn off the 3 settings
- [x] repeat the tests and check that you get the opposite all of the above
- create a new org
- [x] everything OK, can change these settings?
